internal/cloud/cloudplan/saved_plan.go

```diff
@@ -16,13 +22,34 @@ type SavedPlanBookmark struct {
 
 func LoadSavedPlanBookmark(filepath string) (SavedPlanBookmark, error) {
 	bookmark := SavedPlanBookmark{}
-	data, err := os.ReadFile(filepath)
 
+	file, err := os.Open(filepath)
```

Comment:
Do we intend to handle possible symlinks to these bookmarks? @nfagerlund

Reply:
🤔 Do you know, I hadn't even considered it. 😓 

My gut feeling is: **let's don't.** 

- Symlinks are always a can of worms.
- These files should be VERY short-lived and also tiny, so I can't currently imagine any reason to symlink one.
- We can add support later if a use case arises.

Reply:
marking as resolved 👍